app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -1,11 +1,27 @@
-<p><%= t("devise.mailer.invitation_instructions.hello", email: @resource.email) %></p>
+<% current_user = User.find_by_id(@resource.invited_by_id) %>
```

Comment:
Again, ideally this wouldn't be in the view. Makes it impossible to test and difficult to debug should something go wrong(however unlikely that is).